File: api/DTOs/Admin/Orders/OrderItemBriefDto.cs
using api.Entities.Admin.Order;

namespace api.DTOs.Admin.Orders
{
    //change this to: OrderHeader + collection of items
    public class OrderItemBriefDto
    {
        public int OrderId { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string AboutEmployer { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        
        public int OrderItemId { get; set; }
        public string RequireInternalReview { get; set; }
        public int SrNo { get; set; }
        public int ProfessionId { get; set; }
        public string ProfessionName { get; set; }
        public int Quantity { get; set; }
        public bool Ecnr { get; set; }=false;
        public DateTime CompleteBefore { get; set; }
        public JobDescription JobDescription { get; set; }
        public Remuneration Remuneration { get; set; }
        public string Status { get; set; }
        public bool Checked {get; set;}
    }

    
}

File: client/src/app/_dtos/admin/orderItemBriefDto.ts

export interface IOrderItemBriefDto
{
     srNo: number;
     checked: boolean;
     orderNo: number;
     orderDate: Date;
     customerName: string;
     orderItemId: number;
     requireInternalReview: string;
     categoryId: number;
     categoryRef: string;
     categoryName: string;
     categoryRefAndName: string;
     quantity: number;
     status: string;
     assessmentQDesigned: boolean;
     salaryMin: number;
     salaryMax: number;
}

export class OrderItemBriefDto implements IOrderItemBriefDto
{
     srNo = 0;
     checked: boolean = false;
     orderNo = 0;
     orderDate= new Date();
     customerName='';
     orderItemId=0;
     requireInternalReview = '';
     categoryId = 0;
     categoryRef='';
     categoryName = '';
     categoryRefAndName='';
     quantity=0;
     status='';
     assessmentQDesigned = false;
     salaryMax = 0;
     salaryMin = 0;
}